system/blockdev.go

```diff
@@ -119,7 +116,7 @@ func getValidBlockDevices() (valDevs []string) {
 		for _, edev := range excludedevs {
 			if bdev == edev {
 				// skip unsupported devices
-				InfoLog("skipping device '%s', md slaves unsupported", bdev)
+				InfoLog("skipping device '%s', dm slaves unsupported", bdev)
```

Comment:
Thank you for the fix, Petr. 

While at it, may I suggest a different formulation here: "not applicable for dm slaves", because unsupported could be interpreted as something is (still) missing here...

Reply:
Makes sense to me, a second commit is added which changes "unsupported" to "not applicable" in `getValidBlockDevices()`.